docs/docs/pricing/README.md

```diff
@@ -9,137 +9,150 @@ We believe anyone should be able to run simple, low-volume workflows at no cost.
 
-Read more about our plans and pricing options below.
+[Read more about our plans and pricing here](https://pipedream.com/pricing).
 
-[[toc]]
+## Free Tier
 
-## Developer Tier
+You can run up to 3 workflows and add up to 3 connected accounts for up to 100 credits a day for free, as long as they operate under [Pipedream platform limits](/limits/).
 
-**The Developer tier is free**. You can run any number of sources and workflows, as long as they operate under [Pipedream platform limits](/limits/).
+**To lift the 100 credit limit and increase the number of workflows and connected accounts [upgrade to a paid tier](https://pipedream.com/pricing)**.
 
-**To run any number of invocations, for any amount of time, [upgrade to the Professional tier](https://pipedream.com/pricing)**.
+### Free Tier Connected Accounts
 
-### Developer Tier Support options
+Free Tier accounts can connect up to 3 different service accounts like Twitter, Discord, Google Sheets, or any of the thousands of available connections.
+
+### Free Tier Workflow Limitations
+
+Free Tier account workflows are limited to using a maximum of 100 credits per day and a maximum of 3 active workflows.
+
+Additionally, the event history for Free Tier workflows is limited to only the past 100 events.
+
+Upgrading to a paid tier will increase this event history tracking substantially.
+
+### Free Tier Polling Interval Limitations
+
+Free Tier account triggers powered by polling are limited to 5 minute intervals. Paid tiers have an option to polling at a substantially higher frequency. 
+
+### Free Tier Support options
 
 Users on the Developer Tier have access to community support, on [our forum](https://pipedream.com/community) and Slack. [Visit out Support page](https://pipedream.com/support) for more information.
 
-## Professional Tier
+## Paid Tiers
 
-The Professional Tier includes all the features of the Developer Tier. It also comes with the following benefits:
+[Visit our pricing page](https://pipedream.com/pricing) to learn more about our paid plan options.
 
-- **You have no daily invocations or compute time cap. You can run any number of invocations, for any amount of time**.
-- Your event history per workflow increases from {{$site.themeConfig.FREE_INSPECTOR_EVENT_LIMIT}} to {{$site.themeConfig.PAID_INSPECTOR_EVENT_LIMIT}}.
-- You can increase the [max time per execution](/limits/#time-per-execution) to 750 seconds (up from 300 on the Developer Tier).
-- You can request [QPS increases](/limits/#qps-queries-per-second) for specific HTTP endpoints.
-- When using [concurrency and throttling controls](/workflows/concurrency-and-throttling/), you can increase a workflow's queue size up to {{$site.themeConfig.MAX_WORKFLOW_QUEUE_SIZE}} (free users are capped at a queue size of {{$site.themeConfig.DEFAULT_WORKFLOW_QUEUE_SIZE}}).
+All paid plans vary features based on tier, but each paid plan option will:
 
-### Upgrading to the Professional Tier
+* Lift the daily 100 credits limit
```

Comment:
I'd recommend just removing this. There are other benefits to the paid plans and since the pricing page communicates those, let's just push people there.